OMEdit/OMEditLIB/Resources/nls/OMEdit_de.ts

```diff
@@ -4,80 +4,80 @@
 <context>
     <name>AboutOMEditDialog</name>
     <message>
-        <location filename="../../MainWindow.cpp" line="4469"/>
+        <location filename="../../MainWindow.cpp" line="5036"/>
         <source>About %1</source>
         <translation>Über %1</translation>
     </message>
     <message>
-        <location filename="../../MainWindow.cpp" line="4472"/>
+        <location filename="../../MainWindow.cpp" line="5039"/>
         <source>&lt;h2&gt;%1 - %2&lt;/h2&gt;&lt;b&gt;Connected to %3&lt;/b&gt;&lt;br /&gt;&lt;b&gt;Connected to %4&lt;/b&gt;&lt;br /&gt;&lt;br /&gt;Installation path &lt;b&gt;%5&lt;/b&gt;&lt;br /&gt;&lt;br /&gt;Copyright &lt;b&gt;Open Source Modelica Consortium (OSMC)&lt;/b&gt;.&lt;br /&gt;Distributed under OSMC-PL and GPL, see &lt;u&gt;&lt;a href=&quot;http://www.openmodelica.org&quot;&gt;www.openmodelica.org&lt;/a&gt;&lt;/u&gt;.&lt;br /&gt;&lt;em&gt;Compiled without 3D animation support&lt;/em&gt;.</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>AbstractAnimationWindow</name>
     <message>
-        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="191"/>
+        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="195"/>
         <source>Time [s]:</source>
         <translation>Zeit [s]:</translation>
     </message>
     <message>
-        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="199"/>
+        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="203"/>
         <source>Speed:</source>
         <translation>Geschwindigkeit:</translation>
     </message>
     <message>
-        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="218"/>
+        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="222"/>
         <source>Rotate Left</source>
         <translation>Drehe links</translation>
     </message>
     <message>
-        <location filename="../../Animation/AbstractAnimationWindow.cpp" line="219"/>
-        <source>Rotates the camera left</source>
-        <translation>Dreht die Kamera nach links</translation>
```

Comment:
Some of the original text strings are changed. It seems its only possible to update them manually. This should be fixed instead of simply discarding them.

Reply:
This is DE translation. Unfortunately I can't help on this.
What should I do to help?
Maybe you first create a PR with only lupdate files and merge it and after I will tackle french translation files only.

Reply:
See #11239. You can either rebase this PR or can make a new one.